src/components/Disputes/DisputeInfo.js

```diff
@@ -228,12 +236,12 @@ function DisputeHeader({ id, dispute }) {
               />
             )}
           </h1>
-          {transaction && (
+          {dispute?.status !== Status.Voided && transaction ? (
```

Comment:
```suggestion
          {Boolean(dispute?.status !== Status.Voided && transaction) && (
```

And we can remove the ternary `null` afterwards